.snapshots/memviz_test-TestMap

```diff
@@ -1,11 +1,11 @@
 digraph structs {
   node [shape=Mrecord];
-  3:<4key0> -> 1:name;
-  3 [label="<name> structMap |{<f0> id | \"leaf\"} |{<f1> links | {{<4key0> *memviz_test.structMap| <4value0> true}}}  "];
-  5 [label="<name> structMap |{<f0> id | \"leaf2\"} |{<f1> links | map[*memviz_test.structMap]bool\{\}}  "];
+  3 [label="<name> structMap |{<f0> id | \"leaf2\"} |{<f1> links | map[*memviz_test.structMap]bool\{\}}  "];
```

Comment:
This change just happens because maps don't have deterministic output (yet). Can you revert the changes to this file?

Reply:
Hi, done it.